pom.xml

```diff
@@ -81,7 +81,7 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
         <commons.math3.version>3.6.1</commons.math3.version>
-        <graalvm.version>23.0.3</graalvm.version>
+        <graalvm.version>23.0.1</graalvm.version>
```

Comment:
I might have missed that but why do we need to downgrade the `graalvm.version`?

Reply:
good catch, this was likely a merge conflict I missed / clicked incorrectly

Reply:
I was not intentionally downgrading. I am either missing a recent commit that bumped the version or I messed up my merge during a rebase